activity_calendar/models.py

```diff
@@ -70,6 +265,14 @@ def clean_fields(self, exclude=None):
         if self.start_date >= self.end_date:
             errors.update({'start_date': 'Start date must be before the end date'})
 
+        if self.subscriptions_open < self.subscriptions_close:
+            errors.update({'subscriptions_open': 'Subscriptions must open before they can close'})
+
+        if self.subscriptions_open < timezone.timedelta():
```

Comment:
what does `timezone.timedelta()` return?

Reply:
`timezone.timedelta()` (without any parameters) returns a duration of 0 seconds. This statement basically checks for a negative duration.